torchvision/transforms/functional_tensor.py

```diff
@@ -28,7 +28,12 @@ def _get_image_num_channels(img: Tensor) -> int:
 
 
 def vflip(img: Tensor) -> Tensor:
-    """Vertically flip the given the Image Tensor.
+    """PRIVATE METHOD. Vertically flip the given the Image Tensor.
+
+    .. warning::
+
+        Module ``transforms.functional_pil`` is private and should not be used in user application.
```

Comment:
nit: copy-paste error? shouldn't it be `transforms.functional_tensor` instead? The same applies for the other functions in this file